docs/api/api.yml

```diff
@@ -48,40 +46,43 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/bagIndexOk"
-        404:
-          $ref: '#/components/responses/FileNotFound'
+                $ref: "#/components/schemas/bagIndexArrayOk"
+            application/xml:
+              schema:
+                $ref: "#/components/schemas/bagIndexArrayOk"
+        400:
+          description: The query parameter was not supported.
         500:
           $ref: '#/components/responses/Unavailable'
 
   /bag-sequence?contains={UUID}:
     get:
-      summary: Returns the Bag Index metadata for an item from a bag in any bag store.
+      summary: Returns a newline separated list of UUIDs that are part of a bag-sequence.
       description: |
-        An item is either a directory or file in a bag. The item fetched is specified by the item-id:
-
+        The fetched list is specified by the item-id of any bag in the sequence. The items listed in order of their creation date. If the UUID is not part of any sequence an empty list is returned.
```

Comment:
does this mean the parameter UUID is not part of the returned result?

Reply:
Assuming an existing bag-id is used the parameter UUID will be part of the result, which i tried to imply with "any bag in the sequence". The latter part refers to the response in case a non-existing uuid is used, in which case it will not return a 404 but a 200 with an empty response.